setup.py

```diff
@@ -72,22 +72,23 @@
             # XXX: Intersphinx URLs are broken when using non-default inventory
```

Comment:
I saw that [Sphinx 1.4.6 was released](http://www.sphinx-doc.org/en/stable/changes.html#release-1-4-6-released-aug-20-2016), which fixes sphinx-doc/sphinx#2789 which we encountered.

Would you remove the comment and bump the requirement?